src/components/SketchPage.tsx

```diff
@@ -0,0 +1,88 @@
+import type { LineData } from "../utils/types";
+import { GlobalCompositeOperation, LineCap, Tool } from "../utils/enums";
+import { Layer, Line, Stage } from "react-konva";
+import { useRef, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+import { RED_COLOR, WHITE_COLOR } from "../constants/colors";
+
+type SketchPageProps = {
+    selectedTool : Tool;
+};
+
+const SketchPage = ({
+    selectedTool,
+} : SketchPageProps) => {
+    const isDrawing = useRef(false);
+    const [lines, setLines] = useState<LineData[]>([]);
+
+    const handleMouseDown = (event : any) => {
```

Comment:
always try to avoid `any`, wherever you can
```suggestion
    const handleMouseDown = (event : Konva.KonvaEventObject<MouseEvent>) => {
```

Reply:
Sure